### PR TITLE
Feat : Added Cart Page

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

Can you please review my cart page.
Page is not responsive yet, will work on that after submission.
Inline styling is used in some cases where I had to override a particular class property.

Link - https://adoring-hypatia-7042d1.netlify.app/cart.html

Let me know what you think

Thanks

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -10,10 +19,19 @@
...
     background-color: var(--primary-color);
     padding: 3rem 1rem;
     text-align: center;
     color: white;
     border-top-right-radius: 1rem;
+}
+
+.zero-bottom{
+    position: fixed;
+    bottom: 0;
```
Comment (ark2002) [inline styles.css]: image
Hey Shlok, the footer is blocking my view its rather too thick for the footer , maybe tone down the padding to couple of pixels , also its not advisable to have a fixed footer.

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        <div class="card-container" style = "width:40%">
+            <div class="horizontal-badge flex no-wrap">
+                <div class="img-container">
+                    <img src="https://images.unsplash.com/photo-1643313947128-849e4da4fd5b?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2142&q=80"/>
+                </div>
+                <div class="card-header horizontal-card-header grow-1">
+                    <h4 class = "primary-text m2-top">Premium Jacket</h4>
+                    <p class="card-subtitle m1 secondary-text m2-top">$200</p>
+                    <label for="quantity">Quantity:</label>
+                    <input type="number" id="quantity" name="quantity">
```
Comment (ark2002) [inline cart.html]: here you can use the min, max property of the input tag to restrict the user from going -ve in quantity

File: cart.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ark2002) [inline cart.html:6]: could add a meta description to personalize your page more

**Review by ark2002 — CHANGES_REQUESTED** [review]
There can be some changes you should look into it once

File: styles.css (hunk 1 above)
Comment (shlok30) [reply styles.css]: Hey,

Thanks for the advice, I have removed the fixed positioning and used absolute instead.

File: cart.html (hunk 2 above)
Comment (shlok30) [reply cart.html]: Understood, thanks

**Review by sahiba0915 — COMMENTED** [review]
overall code looks clean and good

**Pull request merged**